dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -2650,11 +2605,12 @@ Spdp::SpdpTransport::join_multicast_group(const DCPS::NetworkInterface& nic,
 
   if (joined_interfaces_.count(nic.name()) == 0 && nic.has_ipv4()) {
     if (DCPS::DCPS_debug_level > 3) {
+      ACE_TCHAR buff[256];
+      multicast_address_.addr_to_string(buff, 256);
       ACE_DEBUG((LM_INFO,
                  ACE_TEXT("(%P|%t) Spdp::SpdpTransport::join_multicast_group ")
-                 ACE_TEXT("joining group %C:%hu on %C\n"),
-                 multicast_address_str_.c_str(),
-                 mc_port_,
+                 ACE_TEXT("joining group %s on %C\n"),
+                 all_interfaces ? "all interfaces" : nic.name().c_str(),
```

Comment:
Looks like this line should be using `buff` and the next line is the interface name or "all interfaces".